exercise_15.js

```diff
@@ -0,0 +1,16 @@
+/*
+For every good kata idea there seem to be quite a few bad ones!
+
+In this kata you need to check the provided array (x) for good ideas 'good' and bad ideas 'bad'. If there are one or two good ideas, return 'Publish!', if there are more than 2 return 'I smell a series!'. If there are no good ideas, as is often the case, return 'Fail!'.
+ */
+
+function well(x) {
```

Comment:
one letter argument is a bad practice
formatting is off, because the `const goodCounter` is at the same level as `function`.
Again the kata seems to be to hard for your skill right now.